view/frontend/web/js/view/payment/method-renderer/omise-offsite-tesco-method.js

```diff
@@ -110,19 +118,15 @@ define(
                             storage.get(serviceUrl, false)
                                 .fail(
                                     function (response) {
-                                        errorProcessor.process(response, self.messageContainer);
-                                        fullScreenLoader.stopLoader();
-                                        self.isPlaceOrderActionAllowed(true);
+                                        handlePaymentError(response);
                                     }
                                 )
                                 .done(
                                     function (response) {
                                         if (response) {
                                             redirectOnSuccessAction.execute();
                                         } else {
-                                            errorProcessor.process(response, self.messageContainer);
-                                            fullScreenLoader.stopLoader();
-                                            self.isPlaceOrderActionAllowed(true);
+                                            handlePaymentError(response);
```

Comment:
I'm a little worried that those `self`s might be relevant and are different, but if you've tested and all is OK, my worries may be unjustified